src/posts/2020-10-28-dyslexia-fonts.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: Dyslexia fonts
+description: A quick overview of dyslexia fonts
+category: Assistive technology
+author: Hamsa Harcourt
+further_reading:
+  - url: https://accessibility.huit.harvard.edu/design-readability
+    title: Design for readability
+    source: Havard University
+date: 2020-10-28
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - assistivetechnology
+---
+
+According to Oxford Advanced Learner's Dictionary:
+
+<blockquote>
+  <p>
+    Dyslexia is a slight disorder of the brain that causes difficulty in reading and spelling, for example, but does not affect intelligence.
+  </p>
+</blockquote>
+
+Dyslexia affects about 1 in 5 people worldwide. Its symptoms range in severity from mild to extreme. Since letters, numbers, punctuation, and other characters  are often transposed by Dyslexia, font choice is an important thing to consider.
```

Comment:
```suggestion
Dyslexia affects about 1 in 5 people worldwide. Its symptoms range in severity from mild to extreme. It transposes letters, numbers, punctuation, and other characters, which affects your ability to read. This makes selecting Dyslexic-friendly fonts an important thing to consider.
```

Another heavy edit, but what I want to communicate here is A) what Dyslexia does and B) You can do something about it with your font selection.